GenderPayGap.WebUI/Views/AccountCreation/ConfirmationPage.cshtml

```diff
@@ -26,23 +26,27 @@
     <div class="govuk-grid-column-two-thirds">
         
         <p class="govuk-body">
-            You are now able to add an organisation(s) to your user account and report their gender pay gap data.
+            You are now able to add your employer to your user account and
+            report their gender pay gap information. 
         </p>
         
-        <h2 class="govuk-heading-m">Editing your contact details or closing your account</h2>
+        <h2 class="govuk-heading-m">Editing your contact details or closing your user account</h2>
         
         <p class="govuk-body">
-            You can edit your contact details within Manage My Account.
+            You can edit your contact details within Manage Account.
```

Comment:
I know Claire didn't mention it, but we could link the Manage Account text here to the actual page? We use that pattern elsewhere on the service and finding the Manage Account page isn't entirely clear (it's the second tab on Manage Organisations), so it'd probably help users to navigate the service from here 😄

Reply:
Adding this link would just redirect to the login page since the email confirmation doesn't log you in, probably not worth adding unless we want to change that behaviour

Reply:
Ah yeah ok cool, makes sense!